spec/models/conversion_host_spec.rb

```diff
@@ -439,7 +439,7 @@
   end
 
   context "address validation" do
-    let(:vm) { FactoryBot.create(:host) }
```

Comment:
Vm was a host?  Was this a bad backport or was it committed to master like this?

Reply:
@agrare Can't remember, it was probably my mistake, though.

Reply:
Hm looks like this should have been fixed by https://github.com/ManageIQ/manageiq/pull/18434/files#diff-244db78f3817d010485d040e062dcd43R192

Reply:
Okay, not sure why the cherry-pick didn't pick up on this change but I'm good with this